src/main/java/buddybot/Main.java

```diff
@@ -0,0 +1,31 @@
+package buddybot;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private BuddyBot buddy = new BuddyBot();
+
+    @Override
+    public void start(Stage stage) {
+        try {
+            FXMLLoader myFxmlLoader = new FXMLLoader(Main.class.getResource("/view/MainWindow.fxml"));
+            AnchorPane ap = myFxmlLoader.load();
```

Comment:
can consider writing out ap as anchorpane in small caps, for better readability!